app/helpers/external_logout_helper.rb

```diff
@@ -10,13 +10,9 @@ def external_logout_props
   end
 
   def mount_external_logout
-    container_class = "external_logout"
-    content_tag(:div, nil, :class => container_class) +
-    mount_react_component(
+    react_component(
       'ExternalLogout',
-      ".#{container_class}",
-      external_logout_props.to_json,
-      flatten_data: true
+      external_logout_props
```

Comment:
Can transform into a one liner
```suggestion
      react_component('ExternalLogout', external_logout_props)
```

Reply:
awesome, thanks!